assets/src/scripts/monitoring-location/store/statistics-data.js

```diff
@@ -24,6 +43,19 @@ const setMedianStats = function(statisticsData) {
 const retrieveMedianStatistics = function(siteno) {
     return function(dispatch) {
         return fetchSiteStatistics({siteno, statType: 'median'}).then((stats) => {
+            Object.entries(stats).forEach(parameterStatGroup => {
```

Comment:
I think this may be another candidate for reconsidering how to do this without mutating the contents. But I think it's fine for now.